### PR TITLE
♿ TetraLogical: P0 components accessibility review

### PR DESCRIPTION
**Pull request by TetraLogicalHelpdesk**
Suggested changes and additions based on the TetraLogical accessibility review commissioned by @nainar / @caroqliu

See the related documentation changes proposed at https://github.com/ampproject/amphtml/pull/29609

### CODE REVIEW

File: examples/source/1.components/amp-img.html [hunk 1]
```diff
@@ -81,15 +81,15 @@
...
+  <amp-img src="/static/samples/img/does-not-exist.jpg" width="300" height="100" layout="responsive" alt="a non-existent image">
     <div fallback>offline</div>
   </amp-img>
 
   <!-- ## No script -->
   <!--
     If you want to ensure that users with JavaScript disabled can see your images, you can add an additional `img` element inside a `noscript` tag. This also makes it possible for webcrawlers without JavaScript runtime to discover your images.
   -->
-  <amp-img src="/static/samples/img/amp.jpg" alt="AMP" width="475" height="268" layout="responsive">
+  <amp-img src="/static/samples/img/amp.jpg" width="475" height="268" layout="responsive" alt="AMP">
```
Comment (caroqliu) [inline examples/source/1.components/amp-img.html:92]: For my understanding, why was the `alt` moved to the end?

File: examples/source/1.components/amp-img.html [hunk 2]
```diff
@@ -102,19 +102,21 @@
   <!--
     ## Dynamically changing the src
 
-    You can use `amp-bind` to dynamically change the `src` URL of an `amp-img`.
+    You can use `amp-bind` to dynamically change the `src` URL of an `amp-img`. Remember to also change the `alt` if the content/meaning of the image changes.
```
Comment (caroqliu) [inline examples/source/1.components/amp-img.html:105]: This is great!

File: examples/source/1.components/amp-img.html (hunk 1 above)
Comment (TetraLogicalHelpdesk) [reply examples/source/1.components/amp-img.html:92]: Apologies, this was done purely for code/style consistency with all the other `<amp-img>` elements in this page. No accessibility reason per se.

**Comment by IanPouncey** [comment]
@googlebot I signed it!

**Comment by TetraLogicalHelpdesk** [comment]
@googlebot I signed it!

**Comment by matthiasrohmer** [comment]
@caroqliu, can you please manually verify consent? 🙂

**Comment by TetraLogicalHelpdesk** [comment]
@googlebot I signed it!

**Comment by TetraLogicalHelpdesk** [comment]
I'm not sure why this isn't working. I can see when signed in to https://cla.developers.google.com/clas with the email address associated with this account on Github that it has a nCLA:

> You are covered by the following corporate agreements
> Agreement | Name | Date Signed
> -- | -- | --
> Google Corporate CLA | TetraLogical | Aug 10, 2020 13:44 PDT

If anyone has any ideas about what might be going wrong, please let me know and I'll try to fix this.

**Comment by caroqliu** [comment]
It looks like we need the CLA signed by @patrickhlauke since that is the account that pushed the commits.

**Comment by patrickhlauke** [comment]
Apologies, had to do a re-assignment of commits and force-push a new history. I believe all changes are still in place, but now associated with the correct @TetraLogicalHelpdesk account (at least enough for the CLA to be recognised)

**Comment by patrickhlauke** [comment]
this change also seems to have accidentally picked up some of the earlier commits that came from a merge/update of the branch. not sure if this will cause problems? it's beyond my immediate git knowledge to untangle, I'm afraid...

**Comment by patrickhlauke** [comment]
@googlebot I consent

**Comment by patrickhlauke** [comment]
@caroqliu reminder about this other PR (since https://github.com/ampproject/amphtml/pull/29609 was merged yesterday)

**Comment by TetraLogicalHelpdesk** [comment]
Perfect, thank you @caroqliu

**Pull request merged**